### PR TITLE
libtpms 0.9.4 (new formula)

### PR DESCRIPTION
**Pull request by cvluca**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/libtpms.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  homepage "https://github.com/stefanberger/libtpms"
+  url "https://github.com/stefanberger/libtpms/archive/refs/tags/v0.9.4.tar.gz"
+  sha256 "61d6f947a60686ec98e7cc5861f0999bd6cdaa1fc2b8901b8dc68d1a715b35cf"
+  license "BSD-2-Clause"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "openssl@1.1"
```
Comment (carlocab) [inline Formula/libtpms.rb:12]: Using OpenSSL 3 as you initially did here is probably a better idea.

File: Formula/libtpms.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  def install
+    system "./autogen.sh", "--prefix=#{prefix}", "--with-openssl", "--with-tpm2"
```
Comment (carlocab) [inline Formula/libtpms.rb:15]:
```suggestion
    system "./autogen.sh", *std_configure_args, "--with-openssl", "--with-tpm2"
```

**Review by carlocab — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@cvluca , thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Comment by idesai** [comment]
@cvluca thanks! do you plan to add support for swtpm next :) ?

**Comment by cvluca** [comment]
> @cvluca thanks! do you plan to add support for swtpm next :) ?

@idesai Yes, you can track this PR https://github.com/Homebrew/homebrew-core/pull/100782

**Pull request closed without merging**